src/sentry/grouping/strategies/configurations.py

```diff
@@ -1,12 +1,15 @@
+from typing import Type
```

Comment:
this file is in the typechecker exclude list so anything you're changing here isn't being checked

also you can just use `type[...]` now instead of `Type[...]` so no need for an import here

if you could remove it from the `pyproject.toml` blocklist and fix what fails that would be great!